ios/reactNativeApp.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
new file from xcode v9.3, shall be committed
[Stack overflow](https://stackoverflow.com/questions/49564513/new-file-created-in-xcode-9-3-wsname-xcworkspace-xcshareddata-ideworkspaceche)